packages/sdk-cli/src/models/localRelay/util.test.ts

```diff
@@ -0,0 +1,89 @@
+import * as fsPromises from 'fs/promises';
+import { tmpdir } from 'os';
+import { join } from 'path';
+import { RELAY_PID_FILE_NAME } from './const';
+import { isInt, readJsonFile } from './util';
+
+// Have to do this, because 'fsPromises' __must__ be mocked (which sets all fs methods to jest.fn())
+jest.mock('fs/promises', () => {
+  const actualFs = jest.requireActual('fs/promises') as typeof fsPromises;
+
+  return {
+    ...actualFs,
+    readFile: jest.fn().mockImplementation(actualFs.readFile),
+  };
+});
+
+describe('isInt', () => {
+  describe('false', () => {
+    it('undefined', () => expect(isInt(undefined)).toBe(false));
+    it('null', () => expect(isInt(null)).toBe(false));
+    it('Infinity', () => expect(isInt(Infinity)).toBe(false));
+    it('-Infinity', () => expect(isInt(-Infinity)).toBe(false));
+    it('number string', () => expect(isInt('5')).toBe(false));
+  });
+
+  describe('true', () => {
+    it('number', () => expect(isInt(5)).toBe(true));
+  });
+});
+
+describe('readJsonFile', () => {
```

Comment:
Why use a mock FS for some cases and a real FS for others?

Reply:
The test cases are actually identical, I just lost track of the case names.

Reply:
I tried to `each` -ify them, but the test cases have subtle differences which make this not worth the effort

Reply:
Okay, but why test a mock and real FS separately?

Reply:
Honestly speaking, I don't know. I just feel much safer with having tests using real FS; like, if I were to choose, I'd leave the "real" tests and remove the mock-fs ones. I think of real-fs cases as a test of whether the assumptions I put into the mock-fs tests are correct

Reply:
https://github.com/Fitbit/developer-bridge/pull/176/commits/d08907b3bf6baff82219004f41e50833185d3a61